TYPO3.Flow/Classes/TYPO3/Flow/I18n/Cldr/Exception/InvalidCldrDataException.php

```diff
@@ -18,6 +19,6 @@
  *
  * @api
  */
-class InvalidCldrDataException extends \TYPO3\Flow\I18n\Exception
+class InvalidCldrDataException extends Exception
```

Comment:
Maybe import as `I18n\Exception`, else it could easily be confused with the PHP base Exception class if IDE collapses imports.